File: sdkwork-app-sdk-go/types/follow_stats_vo.go
package types

// 关注统计响应
type FollowStatsVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	FollowingCount int `json:"followingCount"`
	FollowersCount int `json:"followersCount"`
	MutualCount int `json:"mutualCount"`
}

File: sdkwork-app-sdk-csharp/Models/FollowStatsVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class FollowStatsVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public int? FollowingCount { get; set; }
        public int? FollowersCount { get; set; }
        public int? MutualCount { get; set; }
    }
}
